content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -135,14 +148,32 @@ Some default values for mesh shape dimensions have also been changed to be more
 [`2d_shapes`]: https://bevyengine.org/examples/2D%20Rendering/2d-shapes/
 [`3d_shapes`]: https://bevyengine.org/examples/3D%20Rendering/3d-shapes/
 
-### Gizmos
+#### Gizmos
 
 <div class="release-feature-authors">authors: @RobWalt</div>
 
-TODO
+All of the primitives mentioned above can also be rendered with `Gizmos`. For
+this we designed a generic API which should feel very familiar if you have used
+`Gizmos` before. There are two new methods now:
```

Comment:
- What is "primitives mentioned above"? It's probably referencing the table, but the section above this is now the "Meshing" section.
- Who is "we"?
- Linking to `Gizmos` could be nice.

A much shorter and more direct version could be something like this:

```suggestion
Primitives can also be rendered with [`Gizmos`]. There are two new generic methods:
```